static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -14,22 +14,25 @@ const isGoogleWebPreview: () => boolean = () =>
             navigator.userAgent.indexOf('googleweblight') > -1)
     );
 
+let moduleLoadResult = Promise.resolve();
+
 if (!isGoogleWebPreview()) {
-    import(/* webpackMode: "eager" */ 'prebid.js/build/dist/prebid');
+    moduleLoadResult = import('prebid.js/build/dist/prebid');
```

Comment:
@SiAdcock This will now end up in it's own WebPack fragment.

Reply:
@frankie297 Could we add the magic comment for naming the fragment from the documentation?

The fragment is coming out as `graun.38.js` at the moment. We should probably call it Prebid.js.

See [`webpackChunkName`](https://webpack.js.org/api/module-methods/#magic-comments).

Reply:
👍